prog/dftb+/lib_timedep/linrespgrad.F90

```diff
@@ -318,19 +348,19 @@ subroutine LinRespGrad_old(tSpin, this, iAtomStart, grndEigVecs, grndEigVal, scc
       ALLOCATE(symmetries(1))
       symmetries(:) = [ " " ]
     end if
-
+  
```

Comment:
```suggestion

```

Reply:
- That's something that should appear top of the main linresp routine, and will be done if the RS stuff is merged. But generally i,j is occ, ab is virtual, rs are general MO.  
- There is some overhead to store the occ-occ and vir-vir charges in case one does just a spectrum and no gradients. So yes, I should add a flag for doing just occ-virt or all.

Reply:
Do you want to make the occ-virt flag change in this branch, or as a separate pull request afterwards?

Reply:
I will do it in this branch, no need for a seperate pull request. How does it work: I simply push my changes to the current branch and the pull request is updated with the most recent files automatically?

Reply:
Yes, this is exactly, how it should be done. (Pull requests are always pinned to branches, not to specific commits)